airbyte-integrations/connectors/source-jina-ai-reader/metadata.yaml

```diff
@@ -0,0 +1,42 @@
+data:
+  allowedHosts:
+    hosts:
+      - "https://s.jina.ai"
+      - "https://r.jina.ai"
+  registries:
+    oss:
+      enabled: true
+    cloud:
+      enabled: false
+  remoteRegistries:
+    pypi:
+      enabled: true
+      packageName: airbyte-source-jina-ai-reader
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.2.2@sha256:57703de3b4c4204bd68a7b13c9300f8e03c0189bffddaffc796f1da25d2dbea0
+  connectorSubtype: api
+  connectorType: source
+  definitionId: 47077a7f-7ab0-47ee-b428-650396a708c7
+  dockerImageTag: 0.1.0
+  dockerRepository: airbyte/source-jina-ai-reader
+  githubIssueLabel: source-jina-ai-reader
+  icon: jina-ai-reader.svg
+  license: MIT
+  name: Jina AI Reader
+  releaseDate: 2024-06-17
+  releaseStage: alpha
+  supportLevel: community
+  documentationUrl: https://docs.airbyte.com/integrations/sources/jina-ai-reader
+  tags:
+    - language:python
+    - cdk:low-code
+  connectorTestSuitesOptions:
+    - suite: unitTests
+    - suite: acceptanceTests
+      testSecrets:
+        - name: SECRET_SOURCE-JINA-AI-READER__CREDS
+          fileName: config.json
+          secretStore:
+            type: GSM
+            alias: airbyte-connector-testing-secret-store
```

Comment:
Note to self. We'll want to register a secret here for Airbyte-side integration tests.

cc @aaronsteers

Reply:
Logged here: https://github.com/airbytehq/airbyte-internal-issues/issues/8399